src/client/providers/lintProvider.ts

```diff
@@ -98,7 +98,7 @@ export class LintProvider extends vscode.Disposable {
                 return;
             }
             // Exclude files opened by vscode when showing a diff view
-            if (e.uri.scheme === 'git') {
+            if (e.uri.scheme === 'git' || e.uri.scheme === 'showModifications') {
```

Comment:
I assume there's no magical constant defined somewhere that we could use here instead?

Reply:
None that i'm aware of.
But i'll update so it is a little easier to manage.

Reply:
Added a new commit with a constant for the above resources.